gui/gui/src/TGSpeedo.cxx

```diff
@@ -300,6 +315,17 @@ Bool_t TGSpeedo::HandleButton(Event_t *event)
    return kTRUE;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Change the circular buffer size (used for the automatic mean calculation).
```

Comment:
This would be a good place to explain the feature a bit better I think. E.g. mentioning the fact that `SetMeanValue` is ignored if `SetBufferSize` is called with a greater-than-zero argument, how the mean is calculated in this case, etc.

Reply:
You're right, I'll add some explanation